app/models/form_item.rb

```diff
@@ -218,6 +219,10 @@ def top_level?
     depth == 1
   end
 
+  def enabled?
+    !disabled?
+  end
+
   def visible?
     !hidden?
```

Comment:
Maybe we should just define `visible?` as `!hidden? and !disabled?` That would solve a good deal of our nomenclature issues. Then you can just grep for `visible? && enabled?` and replace with `visible?`.